template.yaml.sample

```diff
@@ -51,3 +51,23 @@ Resources:
                 - "s3:PutObject"
                 - "s3:PutObjectAcl"
               Resource: "*"
+
+  RemoteFileToS3GifMaker:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: bin
+      Handler: remote-file-to-s3-function-gif-maker
+      Runtime: go1.x
+      MemorySize: 1024 # MB
```

Comment:
Compared to 512 MB, total duration becomes 50% but the cost difference is only $0.00005 (almost the same)

```
# Duration: 188495.01 ms Billed Duration: 188500 ms	Memory Size: 512 MB	Max Memory Used: 149 MB	Init Duration: 107.23 ms
# Duration: 97550.98 ms Billed Duration: 97600 ms	Memory Size: 1024 MB	Max Memory Used: 161 MB	Init Duration: 124.69 ms
```

```ruby
[10] pry(main)> (188500 / 100) * cost_512
=> 0.0015707705000000001
[11] pry(main)> (97600 / 100) * cost_1024
=> 0.0016266992
```